### PR TITLE
Fix #5 : Pull request template

### LINKED ISSUE #5: Pull request template
Add Pull request template

Resources:
https://github.com/forem/forem/blob/main/.github/PULL_REQUEST_TEMPLATE.md
https://github.com/oppia/oppia-android/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by MohitGupta121**
Fix #5 : Pull request template

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@rt4914 PTAL, Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+_If your PR includes UI changes, please utilize this checklist:_
+- [ ] Semantic HTML implemented?
+- [ ] Keyboard operability supported?
+- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
+- [ ] Color contrast tested?
+
+_For more info, check out the
+[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._
+
+## Added/updated tests?
```
Comment (rt4914) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: Remove this section

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### UI accessibility checklist
+_If your PR includes UI changes, please utilize this checklist:_
+- [ ] Semantic HTML implemented?
+- [ ] Keyboard operability supported?
+- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
+- [ ] Color contrast tested?
+
+_For more info, check out the
+[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._
```
Comment (rt4914) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: Please don't copy things directly from Forem/Oppia. The links for reference only. We don't need this.
Also, get rid of `UI accessibility checklist` for now.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+_We encourage you to keep the code coverage percentage at 80% and above._
+
+- [ ] Yes
+- [ ] No, and this is why: _please replace this line with details on why tests
+      have not been included_
+- [ ] I need help with writing tests
+
+## [optional] Are there any post deployment tasks we need to perform?
+
+## [optional] What gif best describes this PR or how it makes you feel?
```
Comment (rt4914) [inline .github/PULL_REQUEST_TEMPLATE.md:42]: Remove this last open too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@MohitGupta121 thanks for the PR. I have mentioned the required changes.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (MohitGupta121) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (MohitGupta121) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (MohitGupta121) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

**Review by rt4914 — APPROVED** [review]
LGTM. Congratulations on your first PR @MohitGupta121 🚀

**Pull request merged**